test/go_rules/BUILD

```diff
@@ -79,15 +79,3 @@ gentest(
     test_cmd = "$DATA | tee $RESULTS_FILE",
 )
 
-go_library(
-    name = "package_test_lib",
-    srcs = ["package_lib.go"],
-    package = "package_lib",
-)
-
-go_test(
```

Comment:
is this just going away?

Reply:
Yeah in the plugin the import path is defined by the directory it's in and the package is defined in the source code.